<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewer</class>
 <widget class="QDialog" name="LogViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset theme="view-list-text"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topBox">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Log file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="logSelect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="logView">
     <property name="tabStopWidth">
      <number>40</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save to file...</string>
       </property>
       <property name="icon">
        <iconset theme="document-save"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-close"/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
